FRINXio/frinx-frontend
# FD-460 expired transaction error

| Q                        | A 
| ------------------------ | ---
| Fixed Issue?             | `FD-460` 
| Tests Added + Pass?      | Yes
| Any Dependency Changes?  | No


this is little bit complicated to test, but this is how i dit it:

1. `cd packages/frinx-api && yarn build`
2. `cp -r dist/esm/index.js ../../../frinx-gamma/node_modules/@frinx/api/dist/esm/`
3. run frinx-gamma `yarn dev` against .7
4. open browser dev tools to see network; if you see no 422 error in it copy UNICONFIGTXID from cookies and run in postman/insomnia/curl this POST request with cookie UNICONFIGTXID copied from browser
`http://10.19.0.7/api/unistore/operations/uniconfig-manager:close-transaction` (this basically invalidates tx id)
5. when you refresh gamma in browser, you should see 422 errors form server and popup to refresh tx id - which is CORRECT behaviour
